assets/src/application/ui/layout/entityList/withValidFilterState.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import { EntityListFilterBar, EntityListFilterBarProps } from './filterBar';
+
+const withValidFilterState = (FilterBarComponent: React.ReactType): React.FC<EntityListFilterBarProps> => ({
+	filterState,
+	...props
+}) => {
+	return filterState ? <FilterBarComponent filterState={filterState} {...props} /> : null;
```

Comment:
maybe filterstate might have a safe default values and we might not need this check?

Reply:
Better be safe